peopleflow/views/activity.py

```diff
@@ -91,5 +91,5 @@ def activity_delete(event, venue, activity):
             flash("Deleted activity %s" % activity.title)
             db.session.commit()
         return render_redirect(url_for('venue_activity', event=event.id, venue=venue.id), code=303)
-    return render_template('baseframe/delete.html', form=form, title=u"Delete '%s' ?" % (activity.title),
+    return render_template('baseframe/delete.html.jinja2', form=form, title=u"Delete '%s' ?" % (activity.title),
```

Comment:
@jace Do we change this to `render_delete_sqla` since we have moved delete.html.jinja2 template inside theme folders in baseframe

Reply:
You mean, use Baseframe's `render_delete_sqla`? Yes, if it works.